VideoApi/VideoApi.Domain/Participant.cs

```diff
@@ -34,6 +32,23 @@ private Participant()
             ContactEmail = contactEmail;
             ContactTelephone = contactTelephone;
         }
+        
+        public Participant(string name, string firstName, string lastName, string displayName,
```

Comment:
I may have missed this but why was this constructor needed rather than the existing ones. Sorry I think I am being a bit blind here :(

Reply:
I think the participant ref id is not available for the staff member as they have yet to be added to the conference but I am not sure what its used for. Although I wouldn't want to pass null and use an existing constructor.

Reply:
You are right in that but is not being filled in anyway in this constructor so you can just use the previous method. I think I am missing something here though.